BAC0/tasks/Poll.py

```diff
@@ -10,14 +10,16 @@
 
 # --- standard Python modules ---
 import weakref
-
-# --- 3rd party modules ---
-from bacpypes.core import deferred
+import typing as t
 
 # --- this application's modules ---
 from .TaskManager import Task
 from ..core.utils.notes import note_and_log
 
+if t.TYPE_CHECKING:

```

Comment:
needed to prevent circular import: Device -> Point -> Poll -> Device

Reply:
trusting you on this one

Reply:
https://adamj.eu/tech/2021/05/13/python-type-hints-how-to-fix-circular-imports/

Reply:
and to be clear I added the circular import when I added the type hints, wasn't a problem that needed fixing otherwise